components/wifi/esp8266-driver/ESP8266/ESP8266.cpp

```diff
@@ -572,6 +588,8 @@ nsapi_error_t ESP8266::open_tcp(int id, const char *addr, int port, int keepaliv
     static const char *type = "TCP";
     bool done = false;
 
+    ip_info_print(0);
```

Comment:
Isn't this changing a global modem setting?

Doesn't this mean opening a TCP socket will turn it off, breaking UDP?

Isn't it easier to just leave the setting permanently on?

Reply:
Yes, it easier but we have data additional data, so less performance for tcp.

Reply:
I think a bit about that and you are right. We cannot have difrent ESP main configuration for difference socket. Fix added.